bokehjs/src/lib/models/annotations/box_annotation.ts

```diff
@@ -138,17 +134,17 @@ export class BoxAnnotation extends Annotation {
 
     this.define<BoxAnnotation.Props>(({Number, Nullable}) => ({
       top:          [ Nullable(Number), null ],
```

Comment:
Didn't you update these to `Float | Factor` above?

Reply:
This still needs to be fixed. I will do that together with other changes.